src/main.cpp

```diff
@@ -73,10 +76,27 @@
 #define _MAIN_CRASH_SLEEP 2
 #endif
 
+<<<<<<< Updated upstream
 OswAppSwitcher mainAppSwitcher(BUTTON_1, LONG_PRESS, true, true, &main_currentAppIndex);
 OswAppSwitcher watchFaceSwitcher(BUTTON_1, SHORT_PRESS, false, false, &main_watchFaceIndex);
 OswAppSwitcher settingsAppSwitcher(BUTTON_1, SHORT_PRESS, false, false, &main_settingsAppIndex);
 OswAppSwitcher fitnessAppSwitcher(BUTTON_1, SHORT_PRESS, false, false, &main_fitnessAppIndex);
+=======
+OswHal* hal = nullptr;
+// OswAppRuntimeTest *runtimeTest = new OswAppRuntimeTest();
+
+uint16_t mainAppIndex = 0;              // -> wakeup from deep sleep returns to watch face (and allows auto sleep)
+RTC_DATA_ATTR uint16_t watchFaceIndex;  // Will only be initialized after deep sleep inside the setup() ↓
+uint16_t settingsAppIndex = 0;
+uint16_t fitnessAppIndex = 0;
+uint16_t ToolsAppIndex=0;
+
+OswAppSwitcher mainAppSwitcher(BUTTON_1, LONG_PRESS, true, true, &mainAppIndex);
+OswAppSwitcher watchFaceSwitcher(BUTTON_1, SHORT_PRESS, false, false, &watchFaceIndex);
+OswAppSwitcher settingsAppSwitcher(BUTTON_1, SHORT_PRESS, false, false, &settingsAppIndex);
+OswAppSwitcher fitnessAppSwitcher(BUTTON_1, SHORT_PRESS, false, false, &fitnessAppIndex);
+OswAppSwitcher ToolsAppSwitcher(BUTTON_1, SHORT_PRESS, false, false, &ToolsAppIndex);
+>>>>>>> Stashed changes
```

Comment:
Broken code - this are `git` markings, which you'll need to resolve (VSCode should also highlight and warn about this). This completely prevents this from being merged (aside from the other stuff)..